code/modules/modular_computers/file_system/programs/messenger/messenger_program.dm

```diff
@@ -714,6 +714,7 @@
 
 		var/photo_message = signal.data["photo"] ? " (<a href='byond://?src=[REF(src)];choice=[photo_href];skiprefresh=1;target=[REF(chat)]'>Photo Attached</a>)" : ""
 		to_chat(messaged_mob, span_infoplain("[icon2html(computer, messaged_mob)] <b>PDA message from [sender_title], </b>\"[inbound_message]\"[photo_message] [reply]"))
+		messaged_mob.balloon_alert(messaged_mob, "new PDA message")
```

Comment:
I know PDAs already make an sfx in ``ring()``, if that's not enough then maybe a balloon alert *ring* to the player?

Reply:
Good idea - changed it to do that (and made it a part of `ring()`)